### DISCUSSION TITLE
replicating find optimum set-up problem

### DISCUSSION THREAD

**Discussion by EFraats** [simnibs/simnibs#436] [9. Results and analysis]
I'm trying to replicate the optimal setup I calculated using the optimization algorithm in the manual GUI, but I'm noticing differences in the produced electric field and the mean/peak values. What could be causing this? I suspect it might be related to a specific setting. For example, what are the size and thickness of the electrode used by the optimization tool?

**Comment by TorgeW** [comment]
So you run the simnibs optimization for tDCS and now you try to replicate the result in the GUI?

The leadfield calculation uses small round electrodes on the EEG 10-10 electrode positions with a radius of 5 mm and 4 mm thick by default. 
You can change size and shape (be careful so they don't overlap) by using 

https://simnibs.github.io/simnibs/build/html/documentation/sim_struct/tdcsleadfield.html#tdcsleadfield-doc (electrode)
https://simnibs.github.io/simnibs/build/html/documentation/sim_struct/electrode.html#electrode-struct-doc
